2-mandatory/1-writers.js

```diff
@@ -60,8 +60,16 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 function logAllWriters() {
-  // write your code to log all writers here
-};
+  writers.forEach((writer) => {
+    const firstName = writer.firstName
```

Comment:
I don't think you need all these variables you created inside the loop. Your code should look like the example below and should work without the variables
```
function logAllWriters() {
  writers.forEach((writer) => {
    console.log(`Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`);
  })
}
```

Reply:
I agree with Ika. It is much more readable and it is not wrong :)